styles_test.go

```diff
@@ -488,3 +488,105 @@ func TestGetNumFmtID(t *testing.T) {
 	assert.NotEqual(t, id1, id2)
 	assert.NoError(t, f.SaveAs(filepath.Join("test", "TestStyleNumFmt.xlsx")))
 }
+
+func TestGetThemeColor(t *testing.T) {
+	assert.Empty(t, (&File{}).getThemeColor(&xlsxColor{}))
+	f := NewFile()
+	assert.Empty(t, f.getThemeColor(nil))
+	var theme int
+	assert.Equal(t, "FFFFFF", f.getThemeColor(&xlsxColor{Theme: &theme}))
+	assert.Equal(t, "FFFFFF", f.getThemeColor(&xlsxColor{RGB: "FFFFFF"}))
+	assert.Equal(t, "FF8080", f.getThemeColor(&xlsxColor{Indexed: 2, Tint: 0.5}))
+	assert.Empty(t, f.getThemeColor(&xlsxColor{Indexed: len(IndexedColorMapping), Tint: 0.5}))
+}
+
+func TestGetStyle(t *testing.T) {
+	f := NewFile()
+	expected := &Style{
+		Border: []Border{
+			{Type: "left", Color: "0000FF", Style: 3},
+			{Type: "right", Color: "FF0000", Style: 6},
+			{Type: "top", Color: "00FF00", Style: 4},
+			{Type: "bottom", Color: "FFFF00", Style: 5},
+			{Type: "diagonalUp", Color: "A020F0", Style: 7},
+			{Type: "diagonalDown", Color: "A020F0", Style: 7},
+		},
+		Fill: Fill{Type: "gradient", Shading: 16, Color: []string{"0000FF", "00FF00"}},
+		Font: &Font{
+			Bold: true, Italic: true, Underline: "single", Family: "Arial",
+			Size: 8.5, Strike: true, Color: "777777", ColorIndexed: 1, ColorTint: 0.1,
+		},
+		Alignment: &Alignment{
+			Horizontal:      "center",
+			Indent:          1,
+			JustifyLastLine: true,
+			ReadingOrder:    1,
+			RelativeIndent:  1,
+			ShrinkToFit:     true,
+			TextRotation:    180,
+			Vertical:        "center",
+			WrapText:        true,
+		},
+		Protection: &Protection{Hidden: true, Locked: true},
+		NumFmt:     49,
+	}
+	styleID, err := f.NewStyle(expected)
+	assert.NoError(t, err)
+	style, err := f.GetStyle(styleID)
+	assert.NoError(t, err)
+	assert.Equal(t, expected.Border, style.Border)
+	assert.Equal(t, expected.Fill, style.Fill)
+	assert.Equal(t, expected.Font, style.Font)
+	assert.Equal(t, expected.Alignment, style.Alignment)
+	assert.Equal(t, expected.Protection, style.Protection)
+	assert.Equal(t, expected.NumFmt, style.NumFmt)
+
+	expected = &Style{
+		Fill: Fill{Type: "pattern", Pattern: 1, Color: []string{"0000FF"}},
+	}
+	styleID, err = f.NewStyle(expected)
+	assert.NoError(t, err)
+	style, err = f.GetStyle(styleID)
+	assert.NoError(t, err)
+	assert.Equal(t, expected.Fill, style.Fill)
+
+	expected = &Style{NumFmt: 27}
+	styleID, err = f.NewStyle(expected)
+	assert.NoError(t, err)
+	style, err = f.GetStyle(styleID)
+	assert.NoError(t, err)
+	assert.Equal(t, expected.NumFmt, style.NumFmt)
```

Comment:
This unit test was failed. I suggest create a new function `isLangNumFmt` to check if a given number format ID is a built-in language glyphs number format code, note that the ID range is 27-36, 50-62, 67-81.